api/middleware/jwt.go

```diff
@@ -36,8 +37,19 @@ func JwtConfigGenerate(jwtKey, realmName string, db *gorm.DB, l *zap.SugaredLogg
 				lAuth.Warn("bad login")
 				return userId, false
 			}
-			lAuth.Info("successful login")
-			return userId, true
+			// fixes https://github.com/RTradeLtd/Temporal/issues/405
+			// regardless of whether or not they are providing username or email
+			// always return the username
+			usr, err := userManager.FindByUserName(userId)
+			if err != nil {
+				usr, err = userManager.FindByEmail(userId)
```

Comment:
Just wondering. If someone registers a user name that matches my email, will they log in as me?

Reply:
Not unless they know your password, since they still have to sign in before getting to this step at line 31.